pip/qsharp/_native.pyi

```diff
@@ -218,16 +231,28 @@ class Interpreter:
         an operation of a lambda expression. The operation must take only
         qubits or arrays of qubits as parameters.
 
+        :param callable: The callable to synthesize the circuit for.
+
+        :param args: The arguments to pass to the callable.
+
         :raises QSharpError: If there is an error synthesizing the circuit.
         """
         ...
 
-    def estimate(self, entry_expr: str, params: str) -> str:
+    def estimate(
+        self,
+        params: str,
+        entry_expr: Optional[str],
```

Comment:
So maybe entry_expr should have been optional above (prior comment)? Do you provide one or the other? What happens if you provide both?

I think better help docs on these APIs that now take both an entry expression and a callable would be helpful.

Reply:
These help docs aren't part of our public API surface. I'll update them anyway, but this is just for our benefit, not anything for users to see.